### PR TITLE
Custom prosody modules

### PR DESCRIPTION
**Pull request by sapkra**
I need a possibility to add custom prosody modules. I think this is a flexible solution to do that.

FYI: It will have some small conflicts with #29. I have to resolve them after one of the PRs is merged.

### CODE REVIEW

File: prosody/Dockerfile [hunk 1]
```diff
@@ -10,5 +12,4 @@ COPY rootfs/ /
 
 EXPOSE 5222 5269 5347 5280
 
-VOLUME /config
-
+VOLUME ["/config", "$CUSTOM_MODULES"]
```
Comment (saghul) [inline prosody/Dockerfile]: this should be a fixed volume: `/prosody-plugins`

File: prosody/Dockerfile [hunk 2]
```diff
@@ -1,5 +1,7 @@
 FROM jitsi/base
 
+ENV CUSTOM_MODULES=/prosody-plugins-custom
```
Comment (saghul) [inline prosody/Dockerfile]: this is not needed

File: prosody/Dockerfile (hunk 1 above)
Comment (saghul) [reply prosody/Dockerfile]: You can add a variable in the yaml file to device which directory is mounted there

Comment (sapkra) [reply prosody/Dockerfile]: But when we add a fixed volume to `/prosody-plugins` the volume will overwrite the prosody plugins we have added in #29 
So I can change it to be a hardcoded path but it shouldn't be `/prosody-plugins`.
I separated the default and custom modules in different branches so at the end we will have one `/prosody-plugins` and `/prosody-plugins-custom` directory.

Comment (saghul) [reply prosody/Dockerfile]: Sounds good, let's make it `/prosody-plugins-custom` then.

Comment (sapkra) [reply prosody/Dockerfile]: Done

**Comment by saghul** [comment]
Nice work! I'll give it a test and merge.

**Comment by saghul** [comment]
Landed in fb1fdb1d53d805804245be93cc009337b4681727 and 95a55915b7f43312934f391aeda87782366b1689 cheers!

May I please ask you to follow the commit message "convention" we have? Thanks a lot for all your contributions thus far!

**Comment by sapkra** [comment]
Is the convention documented somewhere or should I just learn it by reading the commit messages and understand how they are structured?

**Comment by saghul** [comment]
No it's not documented, but if you look at git log you'll see there is not much to it :-)

**Pull request closed without merging**